app/models/user.rb

```diff
@@ -67,6 +67,7 @@ class User < ApplicationRecord
     s.string :twitter_oauth_token
     s.string :twitter_oauth_token_secret
     s.string :twitter_username
+    s.array :keybase_signatures
```

Comment:
```suggestion
      s.string :keybase_signatures, array: true, default: [], null: false
```

I'm not super-familiar with typedstore so it needs a doublecheck, but I'd prefer to both prohibit nil and be explicit about the type of the elements in the array.

Reply:
From closer reading above I see this isn't an array of strings, it's an array of hashes of symbols => strings... so, yeah, please fix. And please add a test (not roundtripping Keybase, just that we can store + retrieve credentials): some sister sites use postgresql (and I daydream about it...). There's a warning in the typedstore readme about its support for arrays, so at least we could give them a failing test to point directly to the problem.

Reply:
I've never used ruby before, can you elaborate on what I need to do here?

Reply:
I don't know typedstore well enough; I'd just be googling for an example. It's the gem (library) we're using to serialize these attributes to a single database column. We want to configure it with the shape of the data.